### PR TITLE
Remove 1.22 presubmit and periodic lanes

### PR DESCRIPTION
**Pull request by brianmcarey**
Removing the 1.22 lanes as the 1.25 provider has successfully been introduced and the 1.25 lanes are stable.

This is normally covered by an automated PR however the automation is not working due to some lane name changes[1]

This change is also removing the 1.22 sriov lanes as the 1.23 sriov lanes are running and look stable now [2]

[1] https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirt-job-remover/1600480942529449984#1:build-log.txt%3A158 
[2] https://testgrid.k8s.io/kubevirt-periodics#periodic-kubevirt-e2e-k8s-1.23-sriov

/cc @dhiller @xpivarc @enp0s3 

Signed-off-by: Brian Carey <bcarey@redhat.com>

### CODE REVIEW

**Review by dhiller — APPROVED** [review]
/approve

**Comment by dhiller** [comment]
/hold for others to take another look

Please remove the hold if you see fit

**Comment by brianmcarey** [comment]
Opened the following issue for the missing automated PR:  https://github.com/kubevirt/project-infra/issues/2507

**Review by xpivarc — APPROVED** [review]

**Comment by brianmcarey** [comment]
/unhold 

thanks @dhiller @xpivarc

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 1]
```diff
@@ -481,88 +481,6 @@ periodics:
...
-    timeout: 4h0m0s
-  extra_refs:
-  - base_ref: main
-    org: kubevirt
-    repo: kubevirt
-  labels:
-    preset-dind-enabled: "true"
-    preset-docker-mirror-proxy: "true"
-    preset-shared-images: "true"
-  name: periodic-kubevirt-e2e-k8s-1.22-sriov
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:502]: you might want to remove kind-1.22-sriov then as well please

Comment (brianmcarey) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:502]: Yes I was going to check with you and @ormergi before removing the provider from kubevirtci

**Pull request merged**